Entity/BlockInteractor.php

```diff
@@ -160,6 +160,7 @@ public function loadPageBlocks(PageInterface $page)
         $page->disableBlockLazyLoading();
 
         foreach ($blocks as $block) {
+            if ($block->getEnabled() == false) continue;
```

Comment:
- Please add braces
- Please do not compare booleans to booleans. You already have a boolean perfectly usable in your condition : `!$block->getEnabled()`